src/components/tree-item/tree-item.tsx

```diff
@@ -266,11 +275,27 @@ export class TreeItem
           [CSS_UTILITY.rtl]: rtl
         }}
         icon={selectedIcon}
-        scale="s"
+        scale={this.scale === "l" ? "m" : "s"}
       />
     ) : null;
 
     const hidden = !(this.parentExpanded || this.depth === 1);
+    const { hasEndActions } = this;
+    const slotNode = (
+      <slot
+        key="actionsEndSlot"
+        name={SLOTS.actionsEnd}
+        onSlotchange={this.actionsEndSlotChangeHandler}
```

Comment:
should this be `onSlotChange` with a capital C?

Reply:
I think maybe because the event that is fired on an [HTMLSlotElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement) instance when the node(s) contained in that slot change is called `slotchange`, as in one word, that's why we use `onSlotchange`. Which means I should be changing the handler name instead, to match.